README.md

```diff
@@ -8,7 +8,7 @@ You can also extend the agent's performance monitoring to
 [collect and analyze business data](https://docs.newrelic.com/docs/agents/ruby-agent/getting-started/introduction-new-relic-ruby#business-data)
 to help you improve the customer experience and make data-driven business decisions.
 
-The New Relic Ruby Agent is dual-purposed as a either a Gem or a Rails plugin,
+The New Relic Ruby agent is dual-purposed as either a Gem or a Rails plugin,
```

Comment:
`A` -> `a` seems to put us in alignment with the majority of the other agents. Looks like Python's still rocking an `A`. The change makes sense. 👍